drivers/net/macb.c

```diff
@@ -273,47 +283,69 @@ int macb_miiphy_write(struct mii_dev *bus, int phy_adr, int devad, int reg,
 
 	arch_get_mdio_control(bus->name);
 	macb_mdio_write(macb, phy_adr, reg, value);
+	return 0;
+}
 
+static int macb_miiphy_reset(struct mii_dev *bus)
+{
+#if CONFIG_GPIO
```

Comment:
Any way it' not u-boot style - it can be resolved basing on "compatible" and then just checked for GPIO to be valid.

Reply:
My goal is to avoid any changes to DT. This can be implemented in different ways.